guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -11,7 +11,14 @@ There are no highlights with Foreman {ProjectVersion}.
 
 // If this section would be empty otherwise, uncomment the following line:
 ifndef::foreman-deb[]
-There are no upgrade warnings with Foreman {ProjectVersion}.
```

Comment:
For "master", it'd be good to keep this line as a comment. We'll have to basically revert this after branching 3.14.